scvid/module/from_cli.py

```diff
@@ -129,3 +130,18 @@ def __init__(
             perform_mean_correction=perform_mean_correction,
             transform=transform,
         )
+
+
+class TDigestFromCLI(TDigest):
+    """
+    Preset default values for the LightningCLI.
+
+    Args:
+        g_genes: Number of genes.
+        target_count: Target gene epxression count. Default: ``10_000``
+        eps: A value added to the denominator for numerical stability. Default: ``1e-6``
+    """
+
+    def __init__(self, g_genes, target_count: int = 10_000, eps: float = 1e-6) -> None:
```

Comment:
In the future, these args should be replaced with a normalization config YAML (see my earlier comment).